nuxeo-compound-documents/src/main/resources/OSGI-INF/scripted-operations-contrib.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.compound.documents.operations" version="1.0">
+  <documentation>Contribute logic to determinate what kind of compound document we are dealing with</documentation>
+  <extension target="org.nuxeo.automation.scripting.internals.AutomationScriptingComponent" point="operation">
+    <scriptedOperation id="javascript.compound.document.default">
+      <inputType>blob</inputType>
+      <outputType>string</outputType>
+      <category>javascript</category>
+      <description>Specifies what kind of compound document is in a blob.</description>
```

Comment:
Maybe something like _Returns the compound document type based on the given blob_ or something like that, not sure (@troger?)

Reply:
sounds good to me, waiting for the feedback